server/src/main/java/org/opensearch/discovery/InitializeExtensionResponse.java

```diff
@@ -46,18 +49,22 @@
  */
 public class InitializeExtensionResponse extends TransportResponse {
     private String name;
+    private List<String> implementatedInterfaces;
 
-    public InitializeExtensionResponse(String name) {
+    public InitializeExtensionResponse(String name, List<String> implementatedInterfaces) {
         this.name = name;
+        this.implementatedInterfaces = implementatedInterfaces;
     }
 
     public InitializeExtensionResponse(StreamInput in) throws IOException {
         name = in.readString();
+        this.implementatedInterfaces = new ArrayList<String>(Arrays.asList(in.readStringArray()));
```

Comment:
Spelling: implemented

Can't you directly assign `Arrays.asList()` without wrapping it in another list constructor?

Reply:
Yes, I have done the changes.